src/pages/api/webhooks/image-resource-training.ts

```diff
@@ -158,25 +159,32 @@ async function updateRecords(
     },
   });
 
-  if (status === 'InReview') {
-    const model = await dbWrite.model.findFirst({
-      where: { id: modelVersion.modelId },
-      select: {
-        id: true,
-        name: true,
-        user: {
-          select: {
-            email: true,
-            username: true,
-          },
+  const model = await dbWrite.model.findFirst({
+    where: { id: modelVersion.modelId },
+    select: {
+      id: true,
+      name: true,
+      user: {
+        select: {
+          id: true,
+          email: true,
+          username: true,
         },
```

Comment:
I believe we have a reusable selector for users
```suggestion
        select: simpleUserSelect,
```

Reply:
that's missing email so i'll just leave as is for the moment